src/layers/XRLayer/XRLayer.js

```diff
@@ -114,43 +104,30 @@ const defaultProps = {
  */
 const XRLayer = class extends Layer {
   /**
-   * This function chooses a shader (colormapping or not) and
-   * replaces `usampler` with `sampler` if the data is not an unsigned integer
+   * This function replaces `usampler` with `sampler` if the data is not an unsigned integer
+   * and adds a standard ramp function default for DECKGL_PROCESS_INTENSITY.
```

Comment:
```suggestion
   * This function replaces `usampler` with `sampler` if the data is a signed integer
   * and adds a standard ramp function default for DECKGL_PROCESS_INTENSITY.
```

Reply:
This holds for floats as well so "not an unsigned integer" is most general